gulpfile.js

```diff
@@ -25,7 +25,14 @@ gulp.task('ensureDevProxy', function () {
 
 function buildDependenciesArray(dependencies) {
   return Object.keys(dependencies).map(function (dependency) {
-    return dependency + '@' + dependencies[dependency];
+    var version = dependencies[dependency];
+
+    if (version.startsWith('./')) {
```

Comment:
Let's get a comment here. Something like:
`// Module is local, include dependencies from its package.json`